collections/business/general-notifications/event-general-notifications/message-received/event-general-notifications.json

```diff
@@ -45,6 +46,31 @@
           ]
         }
       ]
+    },
+    {
+      "id": "new-message-received-mailout",
+      "name": "New Message Received Mailout",
+      "userPreferenceEnabled": false,
+      "additionalFilterLogic": "#triggeredEvent['origin'] != null && #triggeredEvent['origin'] == 'engagement'",
+      "deliveryOptions": [
+        {
+          "channel": "in-app-notification",
+          "channelSettings": "/new-message-received-mailout/in-app-notification/in-app_channel-settings.json",
```

Comment:
@amorimluzia sorry, this one also should be under `in-app-notification` folder I guess, right? 😅

Reply:
yes, I just added it, seconds ago